client/my-sites/themes/controller/index.web.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Internal Dependencies
+ */
+import SingleSiteComponent from 'my-sites/themes/single-site';
+import MultiSiteComponent from 'my-sites/themes/multi-site';
+import LoggedOutComponent from 'my-sites/themes/logged-out';
+import analytics from 'analytics';
+import i18n from 'lib/mixins/i18n';
+import trackScrollPage from 'lib/track-scroll-page';
+import buildTitle from 'lib/screen-title/utils';
+import { getAnalyticsData } from '../helpers';
+import { setSection } from 'state/ui/actions';
+import { makeElement } from './index.node.js';
+export { details } from './index.node.js';
```

Comment:
Let's place this anywhere that isn't the `import` block, to make it less confusing. Maybe right before the first `export` defined in the module, like `singleSite`.